packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -388,19 +392,29 @@ class AppBarState extends State<AppBar> {
       );
     }
     if (config.actions != null && config.actions.isNotEmpty) {
+      // Expand to at least 48dp if IconButton or PopupMenuButton.
+      List<Widget> sizedActions = config.actions.map<Widget>((Widget action) {
+        return action is IconButton || action is PopupMenuButton ?
```

Comment:
> That's crazy lol. Should we file a bug? Feels like map should be more parallelizable.

Parallelizable in what sense?  The map evaluates the given function over each element of the list in order sequentially.